integration-tests/features/steps/stack_analyses_v2.py

```diff
@@ -255,7 +398,10 @@ def access_url_method(context, url, action, token='without'):
 
 @when('I send {ecosystem} package request with manifest {manifest} '
       'to stack analysis v2 {token} {valid} user key')
-def send_request(context, ecosystem=None, manifest=None, token='without', valid='valid'):
+@when('I send {ecosystem} package request with manifest {manifest} '
```

Comment:
We use general sentence of with / without words for boolean flag in e2e test framework. So its better to have this definition below:
I send {ecosystem} package request with manifest {manifest} to stack analyses v2 {token} {valid} user key {transitive_token} transitives

transitive_token can be 'with' / 'without' and use parse_valid_clause() to parse this flag.